packages/kbn-cli-dev-mode/src/watcher.ts

```diff
@@ -7,44 +7,48 @@
  */
 
 import * as Rx from 'rxjs';
-import {
-  map,
-  tap,
-  takeUntil,
-  count,
-  share,
-  buffer,
-  debounceTime,
-  ignoreElements,
-} from 'rxjs/operators';
-import Chokidar from 'chokidar';
+import Pw from '@parcel/watcher';
+import { REPO_ROOT } from '@kbn/repo-info';
+import { RepoSourceClassifier } from '@kbn/repo-source-classifier';
+import { ImportResolver } from '@kbn/import-resolver';
+import { makeMatcher } from '@kbn/picomatcher';
 
 import { Log } from './log';
 
+const packageMatcher = makeMatcher(['**/*', '!**/.*']);
+
+/**
+ * Any non-package code must match this in order to trigger a restart
+ */
+const nonPackageMatcher = makeMatcher([
+  'src/**',
+  '!src/{dev,fixtures}/**',
+  'x-pack/plugins/**',
```

Comment:
Should we add a note/todo here to remember us that after plugins became packages, this will be redundant/should be changed?

Reply:
Not necessary, I'm very actively working on that PR and this implementation is already updated